pom.xml

```diff
@@ -98,7 +98,8 @@
         <hamcrest.version>1.3</hamcrest.version>
         <jetty.version>9.4.53.v20231009</jetty.version>
         <jersey.version>1.19.4</jersey.version>
-        <jackson.version>2.12.7.20221012</jackson.version>
+        <jackson.version>2.15.3</jackson.version>
+        <core.jackson.version>2.12.7.20221012</core.jackson.version>
```

Comment:
if we plan to go down that route, I would suggest we do it the other way around, upgrade core and make 2.15 the default. Explicitly depend on 2.12 for extensions that cannot upgrade.